page-objects/pages/HomePage.js

```diff
@@ -1,159 +1,161 @@
+class HomePage {
+  openHomePage() {
+    browser.url("https://astage.honesttopaws.com");
+  }
+
+  backPage() {
+    browser.execute(function () {
+      window.history.go(-2);
+    });
+  }
 
+  get cuteCategory() {
+    return $$(".c-category-mix__card")[0];
+  }
 
-class HomePage{
-    openHomePage(){
-        browser.url('https://astage.honesttopaws.com')
-    }
+  get funnyCategory() {
+    return $$(".c-category-mix__card")[1];
+  }
 
-    get cuteCategory(){
-        return $$('.c-category-mix__card')[0]
-        
-    }
+  get goodCategory() {
+    return $$(".c-category-mix__card")[2];
+  }
 
-    get funnyCategory(){
-        return $$('.c-category-mix__card')[1]
-    }
+  get omgCategory() {
+    return $$(".c-category-mix__card")[3];
+  }
 
-    get goodCategory(){
-        return $$('.c-category-mix__card')[2]
-    }
+  get latestArticle1() {
+    return $$(".c-latest__post")[0];
+  }
 
-    get omgCategory(){
-        return $$('.c-category-mix__card')[3]
-    }
+  get latestArticle2() {
+    return $$(".c-latest__post")[1];
+  }
 
-    get latestArticle1(){
-        return $$('.c-latest__post')[0]
-    }
+  get latestArticle3() {
+    return $$(".c-latest__post")[2];
+  }
 
-    get latestArticle2(){
-        return $$('.c-latest__post')[1]
-    }
+  get latestArticle4() {
+    return $$(".c-latest__post")[3];
+  }
 
-    get latestArticle3(){
-        return $$('.c-latest__post')[2]
-    }
+  get editorsPicks1() {
+    return $$(".c-editors-picks__post")[0];
+  }
 
-    get latestArticle4(){
-        return $$('.c-latest__post')[3]
-    }
-    
-    get editorsPicks1(){
-        return $$('.c-editors-picks__post')[0]
-    }
+  get editorsPicks2() {
```

Comment:
Editor Pickes it is an widget adn here you get an post from that widget. Maybe you should change the variable name.